rectools/models/ease.py

```diff
@@ -51,17 +60,31 @@ class EASEModel(ModelBase):
     recommends_for_warm = False
     recommends_for_cold = False
 
+    config_class = EASEModelConfig
+
     def __init__(
         self,
         regularization: float = 500.0,
         num_threads: int = 1,
         verbose: int = 0,
     ):
+        self._config = self._make_config(regularization, num_threads, verbose)
```

Comment:
It seems we don't really need it since we keep both parameters as attributes. Take a look of how it's done for knn model

Reply:
_make_config method was removed.